docs/TutorialReact.md

```diff
@@ -271,6 +273,36 @@ test('CheckboxWithLabel changes the text after click', () => {
 
 The code for this example is available at [examples/enzyme](https://github.com/facebook/jest/tree/master/examples/enzyme).
 
+#### react-testing-library
+
+You have to run `yarn add --dev react-testing-library` to use react-testing-library.
+
+Let's rewrite the test from above using react-testing-library instead of Enzyme.
+
+```js
+// __tests__/CheckboxWithLabel-test.js
+import React from "react";
+import { render, fireEvent, cleanup } from "react-testing-library";
+import CheckboxWithLabel from "../CheckboxWithLabel";
+
+// automatically unmount and cleanup DOM after the test is finished.
+afterEach(cleanup);
+
+test("CheckboxWithLabel changes the text after click", () => {
+  const { queryByLabelText, getByLabelText } = render(
+    <CheckboxWithLabel labelOn="On" labelOff="Off" />
+  );
+
+  expect(queryByLabelText("Off")).toBeTruthy();
+
+  fireEvent.click(getByLabelText("Off"));
+
+  expect(queryByLabelText("On")).toBeTruthy();
+});
+```
+
+The code for this example is available at [examples/enzyme](https://github.com/facebook/jest/tree/master/examples/enzyme).
```

Comment:
This should probably link to `examples/react-testing-library` :)

Reply:
copy pasta fail 🤦‍♂️